doc/api/report.md

```diff
@@ -62,6 +62,26 @@ is provided below for reference.
     "osRelease": "3.10.0-862.el7.x86_64",
     "osVersion": "#1 SMP Wed Mar 21 18:14:51 EDT 2018",
     "osMachine": "x86_64",
+    "osCpus": [
```

Comment:
may be just `Cpus`? in either way, no hard opinion.

Reply:
just keeping with the convention

Reply:
I think `cpus` should be fine.